internal/mapping.go

```diff
@@ -149,7 +153,9 @@ func GetSpannerCols(conv *Conv, srcTable string, srcCols []string) ([]string, er
 // of the following things:
 // a) the new foreign key name is legal
 // b) the new foreign key name doesn't clash with other Spanner
-//    foreign key names
+//
+//	foreign key names
```

Comment:
Why extra lines have been added here ?

Reply:
Cause due to some code formatting it seems. Reverted back to initial state.